p/themes/Fresh-Nord/nord.css

```diff
@@ -318,7 +319,7 @@ a.btn {
 
 /*=== Icons */
 .icon {
-	filter: invert(74%) sepia(29%) saturate(411%) hue-rotate(171deg) brightness(130%) contrast(85%);
+	/*filter: invert(74%) sepia(29%) saturate(411%) hue-rotate(171deg) brightness(130%) contrast(85%);*/
 }
```

Comment:
Is it on purpose?

Reply:
Yeah, because the icons stopped being the right color. I was going to mention that in my previous comment but I forgot...
I was considering using that for "active" buttons so the svg would be the same color as the background, but I couldn't figure it out, so the border will do :P

I would have removed .icon altogether but I wasn't sure

Reply:
To explain I bit more. I originally intended for active buttons to have the same light blue color, and the svg files be the background color via CSS and filter, but I didn't want to bother tbh